changelogs/fragments/502-zfs_bugfix_and_diff_mode_support.yaml

```
@@ -0,0 +1,4 @@
+bugfixes:
+  - zfs - treated received properties as local
```

There's more (see the docs I linked):
```suggestion
  - zfs - treated received properties as local (https://github.com/ansible-collections/community.general/pull/502).
```
Same for the next.

---

I added it as well.